### ISSUE TITLE
Fix 'accessible' prop

### ISSUE THREAD

**Issue by necolas** [necolas/react-native-web#408]
RN's `accessible` prop is better understood as `focusable` - https://facebook.github.io/react-native/docs/accessibility.html#accessible-ios-android. For now, hiding content entirely from AT will require using `aria-hidden` as RN doesn't have an API for that
